Resources/ServerInfo/Guidebook/Mobs/Vox.xml

```diff
@@ -5,12 +5,20 @@
     <GuideEntityEmbed Entity="MobVox" Caption=""/>
   </Box>
 
-  [color=#ffa500]Caution! This species has a severely limiting game mechanic and is not recommended for new players. [/color]
+  [color=#ffa500]Warning! This species is not recommended for new players due to their fatal allergy to oxygen![/color]
 
-  They breathe nitrogen, and [color=#ffa500] oxygen is very toxic to them.[/color]
-  They need to be on internals at all times to avoid being poisoned by the station's oxygen.
-  Eating or drinking will require you to remove your breath mask, although injecting liquid food is an option.
+  Vox breathe nitrogen, and [color=#ffa500] oxygen is toxic to them.[/color]
+  Unfortunately, space stations tend to be full of oxygen, so Vox must use an internal nitrogen supply at almost all
+  times to avoid fatal exposure.
```

Comment:
Same here. Avoid random line breaks in the middle of sentences. Do line breaks between them.